README.md

```diff
@@ -17,5 +23,70 @@ This mod requires MT 5.0.0 and above.
 
 ## Commands
 
-`/watch <player name>`<br>
-`/unwatch` (get back to your initial position)
+All the commands can be modified in settings, here they are listed with their default names.<br>
+
+`/watch <player name>` silently attach to player<br>
+`/unwatch` (get back to your initial position)<br>
+`/watchme <player name>[,<player2 name] ... playerN name]]` invite player(s) to observe caller.<br>
+`/smn` reject an invitation<br>
+`/smy` accept an invitation<br>
+
+## Settings
+
+All settings can be set in minetest.conf or accessed via mod with the global field of same name.
+
+- **spectator_mode.command_accept** (smy)<br>
+ After an invite has successfully been sent, the watcher may accept it with this command.
+- **spectator_mode.command_deny** (smn)<br>
+ After an invite has successfully been sent, the watcher may decline it with this command.
+- **spectator_mode.command_detach** (unwatch)<br>
+To stop observing another player, issue this command.
+- **spectator_mode.extra_observe_privs** ()<br>
+Additional privs granted to observers. e.g. noclip,never_hungry
+- **spectator_mode.extra_observe_privs_moderator** (<spectator_mode.extra_observe_privs>)<br>
+Additional privs granted to observers that used '/watch' command. e.g. jail,kick,teleport<br>
+If left empty, will use spectator_mode.extra_observe_privs
+- **spectator_mode.command_invite** (watchme)<br>
+To invite another player to observe player that issued this command.
+- **spectator_mode.command_attach** (watch)<br>
+To start observing another player, issue this command.
+- **spectator_mode.invitation_timeout** (60)<br>
+Invitations invalidate after this many seconds if they haven't been accepted or denied.
+- *spectator_mode.keep_all_observers_alive*** (false)<br>
+Not only moderators are kept oxygenated, but all observers when this is set to true.<br>
+For auto-feeding the spectator_mode.keep_alive() function needs to be overriden by another mod.
+- **spectator_mode.priv_invite** (interact)<br>
+The priv needed to send observation invites.
+- **spectator_mode.priv_watch** (watch)<br>
+The priv needed to silently observe any player that isn't currently watching another one.
```

Comment:
This causes unnecessary maintenance when new settings are added. I'd simply refer to `settingtypes.txt` directly, and tidy up its formatting so that it's more convenient to read (see Minetest's `settingtypes.txt` as an example).

Reply:
I like that, fixed in: https://github.com/SwissalpS/spectator_mode/commit/b03211e70c827654bf15d3cc28d3f473ed600df8